### DISCUSSION TITLE
AppTrackingTransparency framework iOS

### DISCUSSION THREAD

**Discussion by sachithd** [admob-plus/admob-plus#466] [Q&A]
After submitting the app to the app store I keep getting the following message. Although I have implemented admob.requestTrackingAuthorization() method before showing any ads on the app according to https://admob-plus.github.io/docs/cordova/faq.

Can someone please advise. Thanks

Guideline 2.1 - Information Needed

We're looking forward to completing our review, but we need more information to continue. Your app uses the AppTrackingTransparency framework, but we are unable to locate the App Tracking Transparency permission request when reviewed on iOS 15.1.

Next Steps

Please explain where we can find the App Tracking Transparency permission request in your app. The request should appear before any data is collected that could be used to track the user.

If you've implemented App Tracking Transparency but the permission request is not appearing on devices running the latest OS, please review the available documentation and confirm App Tracking Transparency has been correctly implemented.

If your app does not track users, update your app privacy information in App Store Connect to undeclare tracking. You must have the Account Holder or Admin role to update app privacy information.

Resources

- Tracking is linking data collected from your app with third-party data for advertising purposes, or sharing the collected data with a data broker. Learn more about tracking.
- See Frequently Asked Questions about the new requirements for apps that track users.
- Review developer documentation for App Tracking Transparency.

Since your App Store Connect status is Metadata Rejected, we do NOT require a new binary. To revise the metadata, visit App Store Connect to select your app and revise the desired metadata values. Once you’ve completed all changes, reply to this message in Resolution Center and we will continue the review.

**Comment by boengli** [comment]
If you don't use ATT on the entry page or globally, you need to explain them exactly where/under which navigation point the ATT dialogue appears. And add this to your review notes.

**Reply by sachithd** [reply]
Thanks. I have used it on the entry page. (Globally) and the ads will only be loaded after user confirmation. But still received the Metadata Rejected email. Do you think I still have to add to review notes if I'm using it on app start up?
